File: src/plugin/command/owner/RemoveObjectSpawnCommandPlugin.java
package plugin.command.owner;

import org.redrune.game.entity.actor.player.Player;
import org.redrune.game.content.plugin.type.CommandPlugin;

/**
 * @author Tyluur <itstyluur@gmail.com>
 * @since 9/4/2017
 */
public class RemoveObjectSpawnCommandPlugin extends CommandPlugin {
	
	@Override
	public void handle(Player player, String[] args, boolean console, boolean clientCommand) {
		player.putAttribute("removing_objects", !player.getTemporaryAttribute("removing_objects", false));
		player.getPackets().sendGameMessage("You are now " + (player.getTemporaryAttribute("removing_objects", false) ? "removing" : "examining") + " objects.");
	}
	
	@Override
	public String[] identifiers() {
		return arguments("rmospns");
	}
}

File: src/main/java/plugin/command/owner/RemoveObjectSpawnCommandPlugin.kt
package plugin.command.owner

import game.content.plugin.type.CommandPlugin
import game.entity.actor.player.Player

/**
 * @author Tyluur <itstyluur></itstyluur>@icloud.com>
 * @since 9/4/2017
 */
class RemoveObjectSpawnCommandPlugin : CommandPlugin() {
    override fun handle(player: Player, args: Array<String>, console: Boolean, clientCommand: Boolean) {
        player.putTemporaryAttribute("removing_objects", !player.getTemporaryAttribute("removing_objects", false))
        player.packets.sendMessage(
            "You are now " + (if (player.getTemporaryAttribute(
                    "removing_objects",
                    false
                )
            ) "removing" else "examining") + " objects."
        )
    }

    override fun identifiers(): Array<String> {
        return arguments("rmospns")
    }
}